docs/proposals/kubelet-eviction.md

```diff
@@ -69,6 +69,7 @@ The `kubelet` will support the ability to trigger eviction decisions on the foll
 | Eviction Signal  | Description                                                                     |
 |------------------|---------------------------------------------------------------------------------|
 | memory.available | memory.available := node.status.capacity[memory] - node.stats.memory.workingSet |
+| disk.available   | disk.available := node.stats.fs.available |
```

Comment:
@derekwaynecarr Thanks for the inputs with detail use scenario! Yes, this is my initial concern of having a uniform threshold which lose the flexibility for the cluster admins, eventually makes the feature less useful.